src/components/Modals/Task/TaskModalDataWorker.js

```diff
@@ -45,96 +61,143 @@ class TaskModalDataWorker extends React.PureComponent {
     }
   }
 
-  setTaskDataToState = (data) => {
-    const { ...thisState } = this.state;
-    const { modalTemplate } = this.props;
-    this.setState(
-      {
-        inputsStatus: Object.assign({}, modalTemplate),
-        dataToSend: Object.assign({}, modalTemplate)
-      }
-    );
-    for (let value in data) {
-      if (thisState.hasOwnProperty(value)) {
+  setTaskDataToState = async (data) => {
+    const { modalTemplate, modalType } = this.props;
+
+    await getMembersList().then((membersList) => {
+      this.setState({
+        membersList,
+        inputsStatus: { ...modalTemplate },
+        dataToSend: { ...modalTemplate },
```

Comment:
use better callback method inside `setState` here

Reply:
If I correctly understood you...

```suggestion
this.setState(() => {
        return {
          membersList,
          inputsStatus: modalTemplate,
          dataToSend: modalTemplate,
        };
      });
```

Reply:
and where is prevState ?

Reply:
Fixed